test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.server.cs

```diff
@@ -114,6 +117,189 @@ public void GrpcAspNetCoreInstrumentationAddsCorrectAttributes(bool? enableGrpcA
             Assert.StartsWith("grpc-dotnet", activity.GetTagValue(SemanticConventions.AttributeHttpUserAgent) as string);
         }
 
+        // Tests for v1.21.0 Semantic Conventions for database client calls.
+        // see the spec https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-spans.md
+        // This test emits the new attributes.
+        // This test method can replace the other (old) test method when this library is GA.
+        [Theory]
+        [InlineData(null)]
```

Comment:
Why use a `bool?`? The instrumentation option `EnableGrpcAspNetCoreSupport` is simply a `bool`.

Reply:
That comes from the previous test that I copied (Line 57).

It's being used to test three scenarios in the test setup; explicitly on, explicitly off, and default options.

Reply:
Never mind my above comment. Just realized that the default value set for the variable is `true`.